spec/views/debugging/debugging.html.erb_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe "debugging/debugging", type: :view do
```

Comment:
It would be nce to 'DRY' up this to improve readability as its testing the same page